spec/controllers/relationships_controller_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'spec_helper'
+
+Rspec.describe RelationshipsController do
+  describe "GET index" do
+    context "as an authorized user" do
+      it "sets @relationships to the current user's following relationship" do
+        user = Fabricate(:user)
+        set_current_user(user)
+        other_user = Fabricate(:user)
+        relationship = Fabricate(:relationship, follower: user, leader: other_user)
+        get :index
+        expect(assigns(:leaders)).to eq [other_user]
+      end
+    end
+
+    context "as a guest" do
```

Comment:
I think this is a nice touch. Hmm, though keep in mind that at the moment you require that a user be logged in for all relationship actions. With that in mind, I would add the call to `it_behaves_like "requires sign in"` to all example groups within this controller spec.